src/tests/JNIGridnetClient.java

```diff
@@ -156,24 +195,30 @@ public Response gameStep(int[][] action, int player) throws Exception {
         return response;
     }
 
+    /**
+     * @param player
+     * @return Legal actions mask for given player.
+     * @throws Exception
+     */
     public int[][][] getMasks(int player) throws Exception {
         for (int i = 0; i < masks.length; i++) {
             for (int j = 0; j < masks[0].length; j++) {
-                for (int k = 0; k < masks[0][0].length; k++) {
-                    masks[i][j][k] = 0;
-                }
+                Arrays.fill(masks[i][j], 0);
```

Comment:
may make it easier for the JIT compiler to recognise the code pattern and optimise to use `memset` or SIMD instructions.